reference/5.1/Microsoft.PowerShell.Core/About/about_Type_Operators.md

```diff
@@ -1,27 +1,27 @@
 ---
 keywords: powershell,cmdlet
 locale: en-us
-ms.date: 12/09/2017
+ms.date: 03/20/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_type_operators?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_Type_Operators
 ---
 # About Type Operators
 
 ## SHORT DESCRIPTION
-Describes the operators that work with Microsoft .NET Framework types.
+Describes the operators that work with Microsoft .NET types.
 
 ## LONG DESCRIPTION
 
-The Boolean type operators (-is and -isNot) tell whether an object is an
-instance of a specified .NET Framework type. The -is operator returns a
-value of TRUE if the type matches and a value of FALSE otherwise. The
--isNot operator returns a value of FALSE if the type matches and a value of
-TRUE otherwise.
+The Boolean type operators (`-is` and `-isNot`) tell whether an object is an
+instance of a specified .NET type. The `-is` operator returns a value of TRUE
+if the type matches and a value of FALSE otherwise. The `-isNot` operator
+returns a value of **FALSE** if the type matches and a value of **TRUE**
+otherwise.
```

Comment:
Both sets of TRUE FALSE should be bolded the same.

Reply:
fixed